App/models/topic.py

```diff
@@ -19,11 +21,19 @@ def post_count(self):
         return len(self.posts)
 
     # TODO: Implement observer pattern
-    def subscribe(self, userId):
-        self.text = userId
-
-    def unsubscribe(self, userId):
-        self.text = userId
+    def subscribe(userID,Topicid):
+        createSubscribe = Subscription(userID=userID, Topicid=Topicid)
+        print(f"Subscribed: {userID}")
+        db.session.add(createSubscribe)
+        db.session.commit()
+        return createSubscribe
+
+    def unsubscribe(userID, Topicid):
+        deleteSubscribe = Subscription(userID=userID, Topicid=Topicid)
```

Comment:
this line create a subscritpion object, it should retrieve a subscription object via 
```
sub = Subscription.query.filter_by(topicId=topicId, userId=userId).first()
```